src/utils.rs

```diff
@@ -33,3 +33,8 @@ pub fn derive_test_name(root: &Utf8Path, path: &Utf8Path, test_name: &str) -> St
 
     format!("{}::{}", test_name, relative)
 }
+
+pub fn derive_test_path(root: &Utf8Path, filter: &str, test_name: &str) -> Option<Utf8PathBuf> {
```

Comment:
Could you also add some unit tests for this?

Reply:
Can do

Reply:
Thanks!